### ISSUE TITLE
CLISP broken on Mac

### ISSUE THREAD

**Issue by hraban** [michaelw/cl-dot#42]
Since #39 CLISP is broken on Mac:

```
;; Compiling file /private/tmp/nix-build-system-cl-dot.drv-0/source/package.lisp ...
;; Wrote file /private/tmp/nix-build-system-cl-dot.drv-0/source/package-tmpL4GY2ZJD.fas
;; Compiling file /private/tmp/nix-build-system-cl-dot.drv-0/source/config-graphviz.lisp ...
;; Wrote file /private/tmp/nix-build-system-cl-dot.drv-0/source/config-graphviz-tmpF3F38NB.fas
;; Compiling file /private/tmp/nix-build-system-cl-dot.drv-0/source/attribute.lisp ...
;; Wrote file /private/tmp/nix-build-system-cl-dot.drv-0/source/attribute-tmpIRZ7RSUR.fas
;; Compiling file /private/tmp/nix-build-system-cl-dot.drv-0/source/raw-attributes.lisp ...
;; Wrote file /private/tmp/nix-build-system-cl-dot.drv-0/source/raw-attributes-tmp9JIOZUUB.fas
;; Compiling file /private/tmp/nix-build-system-cl-dot.drv-0/source/attributes.lisp ...
;; Wrote file /private/tmp/nix-build-system-cl-dot.drv-0/source/attributes-tmpB2KTTYYR.fas
;; Compiling file /private/tmp/nix-build-system-cl-dot.drv-0/source/cl-dot.lisp ...
*** - PROBE-FILE: No file name given: #P"/opt/homebrew/bin/"
```

**Comment by rpgoldman** [comment]
OK, I have a fix, but ... ugh: there are no GitHub actions configured to test my branch.  Indeed, there are no tests.  So I'd like to see someone validate my PR on multiple lisps before I merge it.

I simply do not have time to create tests for cl-dot.

**Comment by mdbergmann** [comment]
I can also give it a try on some Lisps.

**Comment by rpgoldman** [comment]
> I can also give it a try on some Lisps.

I would be very grateful if you would.  If you do, please post any problems on the PR, #43

I will put an item on my to-do list to merge this Friday, Feb 16th if no one finds a problem.

**Comment by rpgoldman** [comment]
@hraban Please see if this version fixes your problem. I saw the same issue on clisp (which I don't ordinarily use), and my patch fixed it. But there are a number of different execution paths, and I had neither the ability, time, nor inclination to try them all...

**Comment by hraban** [comment]
Thank you for your time and for the fix @rpgoldman

**Comment by rpgoldman** [comment]
Fixed by #43
